client/geo.py

```diff
@@ -0,0 +1,44 @@
+import re
+import time
+
+import geopy.geocoders
+from django.conf import settings
+
+
+class Locators:
+    NOMINATIM = 'NOMINATIM'
+    GOOGLE = 'GOOGLE'
+    ARCGIS = 3
+
+
+CITY_CLEANUP_RE = re.compile(r"^(.*?)(?: \(.*\))?$")
+
+
+def get_coordinates(city_name, address):
+    city_name = CITY_CLEANUP_RE.match(city_name).group(1)
+    coder = get_geocoder()
+    built_address = "%s, %s, ישראל" % (address, city_name)
+    location = coder.geocode(built_address)
+    # If coding fails, attempt to resolve city_name.
+    if not location:
+        # Required by Nominatim terms of service.
+        if settings.LOCATOR == Locators.NOMINATIM:
+            time.sleep(1)
```

Comment:
I don't know how you currently do this but maybe we want a get_coordinates_bulk function that sleeps between `get_coordinates` calls if Nominatim is used?